doc/serial.md

```diff
@@ -42,3 +43,172 @@ sudo systemctl disable nvgetty
 
 Followed by a reboot.
 
+# New serial protocol
+
+## Configuration
+
+Board A : UART 7
+
+Jetson : UART 2 (pins 8 (TX) and 10 (RX))
+
+230400-8N1 - Little endian
+
+8 bits 230400 baud 1 stop bit & no parity bit
+
+## Protocol
+
+General message layout
+
+|   SOF  | `cmd_id`  | `data_len` | `data ` |
+|--------|-----------|------------|---------|
+| 1 byte |  1 byte  |  1 byte    |         |
```

Comment:
`cmd_id` was 2 bytes before, just to make sure the change to 1 in the doc is voluntary

Reply:
It's been implemented as 1 byte on the cs side as well, so I think we're good with this